<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageImportDialog</class>
 <widget class="QDialog" name="ImageImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Import Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbImportFrom">
     <property name="title">
      <string>Import From</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Import Images &amp;From:</string>
        </property>
        <property name="buddy">
         <cstring>importFolder</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="importFolder"/>
      </item>
      <item>
       <widget class="QPushButton" name="importBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCopyTo">
     <property name="title">
      <string>Copy Images to Another Folder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="copyFiles">
        <property name="text">
         <string>&amp;Copy files from original folder to folder below:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Copy &amp;To:</string>
        </property>
        <property name="buddy">
         <cstring>copyToFolder</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="copyToFolder"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="copyToBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="removeOrig">
        <property name="text">
         <string>Remove Original File (for example, from a flash card)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="renameDuplicates">
        <property name="text">
         <string>If &amp;duplicates, append date and time to make file unique</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbSlideGroup">
     <property name="title">
      <string>Slide Group</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="2" colspan="3">
       <widget class="QRadioButton" name="createNewGroup">
        <property name="text">
         <string>Create &amp;New Slide Group for Images</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QRadioButton" name="addImagesToExisting">
        <property name="text">
         <string>&amp;Add Images to Existing Slide Group</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>New &amp;Group Name:</string>
        </property>
        <property name="buddy">
         <cstring>newGroupName</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="newGroupName"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Add to Grou&amp;p:</string>
        </property>
        <property name="buddy">
         <cstring>existingGroup</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="existingGroup"/>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Slide Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="flagAutoChange">
        <property name="text">
         <string>Automatically change to next photo slides after:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="flagShowNames">
        <property name="text">
         <string>Show File Names on Slide</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="flagNotDefault">
        <property name="text">
         <string>Only if not default camera file name</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="flagLoop">
        <property name="text">
         <string>Automatically loop back to the start of the group when finished</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="changeTime">
        <property name="suffix">
         <string> seconds</string>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="zoomImage">
        <property name="text">
         <string>Turn on Image Smooth-Zoom Effects</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>copyToFolder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>copyToBrowse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>renameDuplicates</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createNewGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createNewGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>newGroupName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addImagesToExisting</sender>
   <signal>toggled(bool)</signal>
   <receiver>existingGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addImagesToExisting</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>removeOrig</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
